scalafix-core/src/main/scala/scalafix/v0/Rule.scala

```diff
@@ -83,7 +83,7 @@ abstract class Rule(ruleName: RuleName) { self =>
     // This overload of apply if purely for convenience
     // Use `applyAndLint` to iterate over Diagnostic without printing to the console
     val result =
-      PatchInternals(patches, ctx, semanticOption)
+      PatchInternals(patches, ctx, semanticOption, suppress = false)
```

Comment:
are changes in this file related to the other one above?

Reply:
no. you are right